README.md

```diff
@@ -1,76 +1,88 @@
 
-0. Prerequesites
+1. Requirements:
+
 - Install [Docker](https://docs.docker.com/install/) and [Docker Compose](https://docs.docker.com/compose/install/). Docker Compose is included when installing Docker on Mac and Windows, but will need to be installed separately on Linux.
+- Install [Node.js](https://nodejs.org/en/)
+- Install [Reaction Platform](https://github.com/reactioncommerce/reaction-platform)
+- Make sure you are not running any applications on ports `3000` and `4000`.
 
-1. Clone the main [Reaction repo](https://github.com/reactioncommerce/reaction) and checkout the `release-1.15.0` branch
-    ```sh
-    git clone git@github.com:reactioncommerce/reaction.git
-    cd reaction
-    git checkout release-1.15.0
+1. Follow the instructions at [Reaction Platform](https://github.com/reactioncommerce/reaction-platform) to run Reaction, Reaction Hydra and the starterkit with Reaction Platform.
 
-    # change directory to the parent of your reaction install
-    cd ..
-    ```
+After the initial bootstrapping, you can use `make start` to start all the necessary containers:
 
-2. Clone this repo
-    ```sh
-    git clone git@github.com:reactioncommerce/reaction-next-starterkit.git
-    ```
+```sh
+make start
+```
+
+1. Once you start the `make` process, you can The Platform will start running the following apps at these URLs:
 
-3. Create a local docker network
+| Application  | URL                     |
+| ------------ | ----------------------- |
+| `starterkit` | localhost:4000          |
+| GraphQL API  | localhost:3000/graphiql |
+| `reaction`   | localhost:3000          |
 
-    You'll need to create a docker network for the GraphQL service and the Reaction Storefront to communicate
-    ```
-    docker network create api.reaction.localhost
-    ```
-    You can run `docker network ls` to verify the network has been created.
+Visit the storefront at `localhost:4000` and the GraphQL API playground at `localhost:3000/graphiql`.
 
-4. Start Reaction's GraphQL server
-    From your `reaction` directory run
-    ```
-    docker-compose up -d --build
-    ```
-    You'll need to run the full app at least once in order for step 5 to work. After this initial run, if you don't want to start the Reaction Meteor app, you can just run `docker-compose up -d devserver`
+1. To view the logs for the Starterkit, run: `docker logs reaction-next-starterkit_web_1 -f`
 
-5. Generate a Meteor login token
+## Configuration
 
-    _This process will be eliminated once we've built out the GraphQL API for authentication_
-    - Visit the Reaction Meteor shop [localhost:3000](http://localhost:3000)
-    - Open devtools and copy the Meteor.loginToken from `localStorage`.
+### Set up Stripe
 
-6. Setup the Storefront environment
+When running the storefront from a new Reaction shop, you will need to configure Stripe payment processing and shipping options to test a complete order checkout flow. After signing up for a Stripe API key, follow these steps:
```

Comment:
lets change: "When running the storefront from a new Reaction shop" to "When running the storefront and Reaction for the first time,"